src/style.css

```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search-input {
+  height: 70px;
+  width: 100%;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  text-indent: 62px;
+  color: rgba(61, 78, 97, 1);
+  background-position: 26px 25px, 0 0;
+  background-image: url(images/Search.svg);
```

Comment:
just add html element img with link to this icon. Then you won't need to copy it here and in search-input:focus